exporters/variables-scss/src/generators/cssGenerator.ts

```diff
@@ -44,6 +59,35 @@ export const tokenToCSSByType = (
     return formatTokenName(name, value);
   }
 
+  if (token.tokenType === TokenType.shadow) {
```

Comment:
Here again.

Reply:
Something like `is/hasTokenType('shadow')`...

Reply:
created function for better readability 👍